hack/update-codegen.sh

```diff
@@ -44,6 +44,13 @@ kube::codegen::gen_client \
   --with-watch \
   "${REPO_ROOT_DIR}/pkg/apis"
 
+kube::codegen::gen_client \
```

Comment:
You can clean up this as in https://github.com/knative/serving/pull/15703/files#diff-149dfe7bb29d1191dceae3a52915e750e64b7f87257a5fb309c29d3056e2a95d. We are removing the generated clientsets in favor of the original ones